stages/org.osbuild.tar

```diff
@@ -123,6 +128,9 @@ def main(inputs, output_dir, options):
     if options.get("xattrs", True):
         extra_args += ["--xattrs", "--xattrs-include", "*"]
 
+    if options.get("sparse", False):
+        extra_args += ["--sparse"]
```

Comment:
Looking for some input as to whether defaulting `sparse` to `true` would be a good idea?

Reply:
Since this is a change to an existing stage I'd say no, leave it False by default to match existing behavior.

Reply:
yeah. i was trying to think of a reason someone would want not sparse?

That's the only reason I would consider just making it the default.

Reply:
For example, when we [create the GCP image](https://github.com/osbuild/images/blob/985caf2b26cf4d36b6e7e9c4fb51c19149900e40/pkg/image/disk.go#L101-L113) why would we want the single file in the tar archive to not be sparse?

Of course, the archive is then gzipped which probably means the actual tar.gz is probably not that much larger than it would be if the file wasn't sparse, but I think it's still worth having the conversation.